### ISSUE TITLE
Please help me

### ISSUE THREAD

**Issue by Deerzh** [AlphaJia/pytorch-faster-rcnn#19]
when I run a command which Train with mobilenet, I got information like this:
Epoch: [0]  [ 2950/19715]  eta: 1:36:08.972184  loss_rpn_box_reg: 0.8739 (1.0911)  lr: 0.005000  loss: 1.6920 (2.0218)  loss_classifier: 0.3552 (0.4206)  loss_box_reg: 0.1971 (0.2049)  loss_objectness: 0.2848 (0.3052)  time: 0.3408  data: 0.0152  max mem: 0

 I want to know the number 19715 represent what and can you tell me how to reduce it's value, because I want to finish the train process as soon as possible.

**Comment by vilalaoil** [comment]
19715 respent the number of  photo in your     data_root_dir .
to finish the train faster, you can use GPU, more GPU or reduce your batch_size in config/train_config.py whie its accuracy may be down
